tracer/test/snapshots/Security.AspNetMvc5.Classic.enableSecurity=True.__test=server.request.body_url=_Home_Upload_body={-Property1-- -[$slice]-}.verified.txt

```diff
@@ -31,14 +32,17 @@
       actor.ip: 86.242.244.246,
       appsec.event: true,
       env: integration_tests,
+      http.client_ip: 127.0.0.1,
       http.method: POST,
       http.request.headers.content-length: 25,
       http.request.headers.content-type: application/json; charset=utf-8,
       http.request.headers.host: localhost:00000,
       http.request.headers.user-agent: Mistake Not...,
       http.request.headers.x-forwarded: 86.242.244.246,
+      http.route: /home/upload,
```

Comment:
I expected this to be `{controller}/{action}/{id}`, do you know what's happening here?